retriever/lib/download.py

```diff
@@ -23,9 +20,6 @@ def download(dataset, path='./', quiet=False, sub_dir='', debug=False, use_cache
     engine.use_cache = use_cache
 
     script_list = SCRIPT_LIST()
-    if not script_list or not os.listdir(SCRIPT_WRITE_PATH):
-        check_for_updates()
-        script_list = SCRIPT_LIST()
     scripts = name_matches(script_list, args['dataset'])
```

Comment:
Should not download all the scripts if no offline script is present.

Reply:
same as above.